Procfile

```diff
@@ -38,8 +38,10 @@ celery_scheduler: REMAP_SIGTERM=SIGQUIT newrelic-admin run-program celery beat -
 # TODO: Merge these two listeners into one since they use so little CPU each (bug 1530965).
 pulse_listener_pushes: newrelic-admin run-program ./manage.py pulse_listener_pushes
 pulse_listener_tasks: newrelic-admin run-program ./manage.py pulse_listener_tasks
+#Two listeners are combined 
```

Comment:
Chang this to `# Both Pulse listeners; for now, do not run at the same time as the othe ones.`